lib/src/base-styles/_nx-colors.scss

```diff
@@ -92,6 +92,21 @@
   --nx-color-counter: var(--nx-swatch-grey-05);
   --nx-color-counter-active-text: var(--nx-swatch-white);
 
+  //NxProgressBar
+  --nx-color-progress-bar-background: var(--nx-swatch-indigo-95);
+  --nx-color-progress-bar-progress: var(--nx-swatch-teal-60);
+  --nx-color-progress-bar-text-success: var(--nx-swatch-green-25);
+  --nx-color-progress-bar-icon-success: var(--nx-color-progress-bar-text-success);
+  --nx-color-progress-bar-text-error: var(--nx-swatch-red-40);
+  --nx-color-progress-bar-icon-error: var(--nx-color-progress-bar-text-error);
```

Comment:
Eliminate this variable and use `--nx-color-validation-invalid-icon` directly in NxProgressBar

Reply:
> Eliminate this variable and use `--nx-color-validation-invalid-icon` directly in NxProgressBar

The error icon appears when the progress bar hits an error. Is `--nx-color-validation-invalid-icon` an accurate enough variable name for a variable being applied to an error icon? "Invalid" and "error" are similar, but not the same.

Reply:
> The error icon appears when the progress bar hits an error. Is `--nx-color-validation-invalid-icon` an accurate enough variable name for a variable being applied to an error icon? "Invalid" and "error" are similar, but not the same.

For this and the one below, I personally think we should use the `--nx-color-validation-invalid/valid-icon` to save variables and to me the meanings are function of the icons are quite similar. However, you do bring up a good point, so you could bring it up in WW tomorrow to get others' thoughts on the matter.

Reply:
> > The error icon appears when the progress bar hits an error. Is `--nx-color-validation-invalid-icon` an accurate enough variable name for a variable being applied to an error icon? "Invalid" and "error" are similar, but not the same.
> 
> For this and the one below, I personally think we should use the `--nx-color-validation-invalid/valid-icon` to save variables and the function of the icons are quite similar. However, you bring up a good point, so maybe you could bring it up in WW tomorrow to get others' thoughts on the matter?

Per the WW conversation, it was decided that we should use new variables over `--nx-color-validation-invalid/valid-icon`. The new variables have more specific names for each CSS property. i.e.
Below is a more specific name:
```
.nx-icon {
      color: var(--nx-color-progress-bar-icon-success);
    }
```
Than
```
.nx-icon {
      color: var(--nx-color-validation-valid);
    }
```
I believe using new variables would make the selectors easier to read.

Reply:
I think this conversation is resolved so I'm closing it. If you want to talk about this more, feel free to open it back up.